README.md

```diff
@@ -1,70 +1,234 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command in your terminal (inside the project folder):
+
+- npm init
+- npm install
+- npm start
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run Tests <a name="run-tests"></a>
+
+To run the the tests, you will have to run the following command in the console (inside the project folder): 
+
+- npm install
+
+And then you will have to run the following command in the console (inside the project folder): 
+
+- npm test
+
+You will see the outcome in the console
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Deployment <a name="deployment"></a>
+
+To deploy this project in a local enviroment, you will have to run:
+
+- npm start
+
+Then, you will see the project run in a local host: 3000.
+Otherwise, you can see the project deployed in Render in the live demo.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## Author <a name="authors">Vitoesi Elijah</a>
+
+  - GitHub: [@Ellyboi](https://github.com/Ellyboi)
+  - Twitter: [@Elijahvitoesi](https://twitter.com/Elijahvitoesi)
+  - LinkedIn: [Vitoesi Elijah](https://www.linkedin.com/in/vitoesi-elijah-61961611a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## Future Features <a name="future-features"></a>
+
+
+- [ ] **[Finishing the full website]**
+- [ ] **[Adding more cryptos]**
+- [ ] **[Creating new pages (about, contact, etc)]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- Project Presentation -->
+
+## :movie_camera: Project Presentation <a name="project-presentation"></a>
+
+- [Presentation](Country's Details App - 25 May 2023 - [https://www.loom.com/share/df65b6d1292846c88af2370c5ea6f109]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](). Or send them directly to me.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ️ Show your support <a name="support"></a>
+
+If you like this project feel free to let me know via Linkedin - Github - Twitter, also don't forget to leave your ️. I will always appreciate your comments.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## Acknowledgments <a name="acknowledgements"></a>
+
+- Templete used in project provided [Nelson Sakwa](https://www.behance.net/sakwadesignstudio).
+- API used for the data [Air pollution](https://date.nager.at/api/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## License <a name="license"></a>
+
+This project is [MIT](./LICENSE(https://github.com/Ellyboi/react-capstone-project/blob/metricWebApp/LICENSE)) licensed.
```

Comment:
- [ ] Kindly add a correct MIT License link to your README. It is broken at this moment.

![license](https://github.com/Ellyboi/react-capstone-project/assets/26597432/dcd7f512-c0e9-49e1-9efa-cb7148c65628)